tests/kafkatest/tests/streams/streams_broker_down_resilience_test.py

```diff
@@ -45,7 +45,7 @@ def setUp(self):
             self.zk.start()
 
     @cluster(num_nodes=7)
-    @matrix(metadata_quorum=[quorum.isolated_kraft], use_new_coordinator=[True, False])
+    @matrix(metadata_quorum=[quorum.combined_kraft], use_new_coordinator=[True, False])
```

Comment:
Why this change? Similar below.

Guess it raises a broader question: should we run with combined or isolate mode? IIRC, at some point in the past when we added KRaft support, we did mix it up on purpose to test both setups.

Frankly not sure, what we should do 🤔

Reply:
My apologies!   I should have explained my reasoning up front.   I updated the `metadata_quorum` based on (this information in kafka.py)[https://github.com/apache/kafka/blob/trunk/tests/kafkatest/services/kafka/kafka.py#L212-L235] In all of our system tests, we're using a combined cluster (not using dedicated nodes for the controller role) so I felt it best to use the value that represented what the test uses.  I could have been wrong in this approach/assumption so I'm willing to revert where necessary